app/views/govuk_publishing_components/components/_layout_super_navigation_header.html.erb

```diff
@@ -217,6 +231,13 @@
         hidden
         id="super-search-menu-toggle"
         type="button"
+       data-ga4="<%= {
```

Comment:
Is the indentation here slightly wrong?

Reply:
I think so - this has exposed how reliant I am on `standardx` to make my JS look good - without it I'm sloppy!